editor/editor/mode/visual.js

```diff
@@ -20,11 +20,25 @@ function Blocks( { blocks, onChange } ) {
 	return (
 		<div className="editor-mode-visual">
 			<div>
-				{ blocks.map( ( block, index ) =>
-					<div key={ index }>
-						{ wp.blocks.getBlockSettings( block.blockType ).edit( block.attributes, onChangeBlock( index ) ) }
-					</div>
-				) }
```

Comment:
> and only assigns a save implementation? 

Yep, could be good for separators and stuff like that.

> We currently allow a block to register itself with neither edit nor save implementations, which will throw if left unchecked

I think we should add some validation to the `registerBlock` (category, save, title, icon are all mandatory to me) (but granted this is a separate task)